modules/caddytls/distributedstek/distributedstek.go

```diff
@@ -145,7 +145,12 @@ func (s *Provider) storeSTEK(dstek distributedSTEK) error {
 // current STEK is outdated (NextRotation time is in the past),
 // then it is rotated and persisted. The resulting STEK is returned.
 func (s *Provider) getSTEK() (distributedSTEK, error) {
-	s.storage.Lock(s.ctx, stekLockName)
+	err := s.storage.Lock(s.ctx, stekLockName)
+	if err != nil {
+		return distributedSTEK{}, fmt.Errorf("failed to acquire storage lock: %w", err)
```

Comment:
```suggestion
		return distributedSTEK{}, fmt.Errorf("failed to acquire storage lock: %v", err)
```

Reply:
Why not wrap it? IMO errors are much more useful when they can be unwrapped and `errors.Is` can be used on it...

Reply:
Because then it exposes that error value to the exported API, and I don't want us or anyone to rely on those unless we have a specific reason to expose them. It adds to our maintenance burden.